pandas/tests/io/pytables/test_complex.py

```diff
@@ -0,0 +1,205 @@
+from distutils.version import LooseVersion
+from warnings import catch_warnings
+
+import numpy as np
+import pytest
+
+import pandas as pd
+from pandas import DataFrame, Series
+from pandas.tests.io.pytables.common import ensure_clean_path, ensure_clean_store
+import pandas.util.testing as tm
+from pandas.util.testing import assert_frame_equal
+
+from pandas.io.pytables import read_hdf
+
+# TODO:
```

Comment:
can you move these tables imports and mods (lines 15-22) from these individual files and put in common instead.

that way we only do these one and skip appropriately.

Reply:
@jreback  I _think_ pytest.importorskip("tables") should be in each module and common.py should be pytest agnostic.

but that does mean duplication of parameter setting unless we add that to the module scoped fixture.